CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
-1.5.4 (Next)
+2.0.0 (Next)
```

Comment:
Bump version.rb or undo. 

Since the next version is actually compatible with newer mongoid's without changes, and the only "breaking" change is that we drop *testing* or support on 1.9.3, I would just leave it as 1.5.4, but up to you.

Reply:
Right, the next version will probably be compatible with 1.9.3. What version would you prefer? `1.6.0` or you still prefer `1.5.4`?

Reply:
I've picked up `1.6.0` to follow semver, but let me know if you have any other preferences.

Reply:
That works!